.circleci/config.yml

```diff
@@ -49,18 +49,6 @@ commands:
   setup-aws-ecr:
     description: 'Install aws cli and login to public and private ECR'
     steps:
-      - run:
```

Comment:
Removed because I was getting errors like:

> Found preexisting AWS CLI installation: /usr/local/aws-cli/v2/current. Please rerun install script with --update flag.